Bindings/Python/examples/posthoc_StatesTrajectory_example.py

```diff
@@ -0,0 +1,81 @@
+# ----------------------------------------------------------------------- #
+# The OpenSim API is a toolkit for musculoskeletal modeling and           #
+# simulation. See http://opensim.stanford.edu and the NOTICE file         #
+# for more information. OpenSim is developed at Stanford University       #
+# and supported by the US National Institutes of Health (U54 GM072970,    #
+# R24 HD065690) and by DARPA through the Warrior Web program.             #
+#                                                                         #
+# Copyright (c) 2005-2018 Stanford University and the Authors             #
+# Author(s): Christopher Dembia                                           #
+#                                                                         #
+# Licensed under the Apache License, Version 2.0 (the "License");         #
+# you may not use this file except in compliance with the License.        #
+# You may obtain a copy of the License at                                 #
+# http://www.apache.org/licenses/LICENSE-2.0.                             #
+#                                                                         #
+# Unless required by applicable law or agreed to in writing, software     #
+# distributed under the License is distributed on an "AS IS" BASIS,       #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         #
+# implied. See the License for the specific language governing            #
+# permissions and limitations under the License.                          #
+# ----------------------------------------------------------------------- #
+
+# This example shows how to use the StatesTrajectory to analyze a simulation
+# (computing joint reaction forces).
+
+import opensim as osim
+import math
+
+model = osim.Model()
+model.setName('model')
+
+# Create a pendulum model.
+# ------------------------
+# In the default pose, the system is a mass hanging 1 meter down from a hinge.
+body = osim.Body('body', 1.0, osim.Vec3(0), osim.Inertia(1))
+model.addComponent(body)
+
+joint = osim.PinJoint('joint',
+        model.getGround(), osim.Vec3(0), osim.Vec3(0), 
+        body, osim.Vec3(0, 1.0, 0), osim.Vec3(0))
+joint.updCoordinate().setName('q')
+model.addComponent(joint)
+
+# This reporter will collect states throughout the simulation at intervals of
+# 0.05 seconds.
+rep = osim.StatesTrajectoryReporter()
```

Comment:
minor, but I would prefer `reporter` instead of `rep` especially when you use variable names like `model`, `body`, `joint`, etc...